README.md

```diff
@@ -1,5 +1,9 @@
 # AWS App Mesh
 
+**NOTE**  
```

Comment:
would it be better to keep this under Getting started

We currently have "China Regions" specific workaround mentioned.  We can make a section about ARM support?